src/sdk/account/account.service.ts

```diff
@@ -232,15 +232,15 @@ export class AccountService extends Service {
     return result;
   }
 
-  async getAccountBalances(account: string, tokens: string[], ChainId: number): Promise<AccountBalances> {
+  async getAccountBalances(account: string, tokens: string[], ChainId: number, provider: string): Promise<AccountBalances> {
```

Comment:
Provider argument must be optional.

Reply:
Otherwise, this will result in a breaking change! Our SDK must always be backward compatible.

Reply:
Yes. It is and I have tested both scenarios in local